index.d.ts

```diff
@@ -86,6 +86,7 @@ declare module 'warframe-items' {
         sex?: Sex;
         sprint?: number;
         passiveDescription?: string;
+	areaAttack?: AreaAttack;
```

Comment:
this line looks differently indented.... could you make sure it's consistent?

Reply:
Yah, looks like 1-space for the unified diff prefix and 8-spaces for the original code, and mine is the `+` prefix plus a hard-tab (dang vim defaults). Guess I need to expand tabs into spaces for this code-base.

Reply:
yeah, sorry about that.

Reply:
Not a problem, I didn't catch that the code doesn't have hard-tabs.  I will adjust my editor for this repository.